.github/workflows/staging-build-rpm.yml

```diff
@@ -334,7 +334,29 @@ jobs:
           .github/scripts/setup_runners_service.sh rpm --es
           export PATH=$JAVA_HOME:$PATH; cd $GITHUB_WORKSPACE/../anomaly-detection; pwd
           ./gradlew :integTestRunner --tests "com.amazon.opendistroforelasticsearch.ad.rest.*IT" --tests "com.amazon.opendistroforelasticsearch.ad.e2e.*IT" -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin
-
+  
+  Test-ALERTING:
```

Comment:
Hello. Please put the new self host runner in provision and cleanup, and also add the needs of this into cleanup.

Reply:
good point.  @gaiksaya, it seems that you just forgot to add them. I see the logic is already in the testing workflow file.

Reply:
Aah! My bad let me add

Reply:
Added!